paddle/fluid/operators/print_op.cc

```diff
@@ -113,25 +127,31 @@ struct Formater {
     } else if (framework::IsType<const bool>(dtype)) {
       Display<bool>(size);
     } else {
-      logs << "\tdata: unprintable type: " << dtype.name() << std::endl;
+      logs << "  - data: unprintable type: " << dtype.name() << std::endl;
     }
   }
 
   template <typename T>
   void Display(size_t size) {
     auto *d = reinterpret_cast<T *>(data);
-    logs << "\tdata: ";
+    logs << "  - data: [";
     if (summarize != -1) {
       summarize = std::min(size, (size_t)summarize);
-      for (int i = 0; i < summarize; i++) {
-        logs << d[i] << ",";
+      if (summarize > 0) {
+        logs << d[0];
+        for (int i = 1; i < summarize; i++) {
```

Comment:
Very tiny issue, not important at all. You could change in the future PR or here if you have time. It's also OK if you don't change because I saw it was just in old code.

Prefer ++i than i++. If you leant C/C++ very early, you may get used to i++, it doesn't cause speed issues at most time but in some extreme cases such as competition, ++i makes difference ...

It's also a code style preference. Ref:
https://google.github.io/styleguide/cppguide.html#Preincrement_and_Predecrement

Reply:
done, thx